### PR TITLE
Ftr sabotage

### PR DESCRIPTION
**Pull request by DanRowe**
sabotaged all our functions in p1! This should fail all tests. Merge #90 first to see them all fail.

### CODE REVIEW

**Review by sagars729 — APPROVED** [review]
Looks good for now,

12 functions have been changed and evenly distributed across 4 people. You might want to wait to see that 12 checks are failing and your compile check is still passing (i.e. merge FTR-actions before this one) 

@willspencer805 @jmablanco you are graded individually on whether you reviewed PRs. Please do that for the next FTR and maybe have two reviewers for each feature-item so you can both get some partial credit back for the features you didn't review.

**Comment by DanRowe** [comment]
waiting on #93

File: Projects/P1/src/Map.java [hunk 1]
```diff
@@ -64,37 +64,33 @@ public boolean move(String name, Location loc, Type type) {
 		
 		comp.setLocation(loc.x, loc.y);
 		components.replace(name, comp);
-
-		//need to update field like how they did in Map.add()
-		if (!field.containsKey(loc)) field.put(loc, new HashSet<Type>());
-		field.get(loc).add(type);
 		
 		return true;
```
Comment (DanRowe) [inline Projects/P1/src/Map.java:68]: Return false to fail test

Comment (jmablanco) [reply Projects/P1/src/Map.java:68]: o ya sorry I didn't really edit it much huh, 1 sec

Comment (jmablanco) [reply Projects/P1/src/Map.java:68]: #94 is the pull request going from sabotage-mapmove to FTR-sabotage, someone PR it please, I assigned @willspencer805

**Comment by jmablanco** [comment]
Going to merge this pull request now, everything but the build test is failing so the sabotage is a success.

**Pull request merged**